src/mscorlib/corefx/System/Globalization/CompareInfo.cs

```diff
@@ -380,12 +380,18 @@ public unsafe virtual bool IsPrefix(String source, String prefix, CompareOptions
             }
             Contract.EndContractBlock();
             int prefixLen = prefix.Length;
+            int sourceLength = source.Length;
```

Comment:
Nit: should this be moved down to where it's used?  We don't need to access Length if prefixLen is 0.  It also doesn't look like it's used in more than one place, so is the local even necessary?